overlord/fdestate/backend/reseal.go

```diff
@@ -70,6 +70,17 @@ type SealingParameters struct {
 	TpmPCRProfile []byte
 }
 
+// EncryptedDisk gives information on the role, path and path to extra
+// legacy keys.
+type EncryptedDisk interface {
+	// GetRole gives the container role of the disk. See KeyslotRoleInfo.Parameters.
+	GetRole() string
+	// GetDevPath gives the path to the device node. This should be the same as the path used for keyring.
+	GetDevPath() string
+	// GetLegacyKeys gives path the legacy keys index by the key name that matches the token key
```

Comment:
would this be more appropriate?
```suggestion
	// GetLegacyKeys gives path of the legacy keys indexed by the key names used in the tokens
```

Reply:
Done.